apps/web/components/auth/nextauth-auth-form.tsx

```diff
@@ -20,37 +13,36 @@ export function NextAuthAuthForm({
   providers,
 }: AuthFormProps) {
   return (
-    <div className="flex flex-col items-center justify-center min-h-[450px] w-full px-4 gap-6">
-      <Card className="w-full max-w-md mx-auto shadow-lg border-2">
-        <CardHeader className="space-y-3 text-center pb-8">
-          <div className="space-y-2">
-            <CardTitle className="text-2xl font-bold tracking-tight">
-              Welcome to Tambo
-            </CardTitle>
-            <CardDescription className="text-base">
-              Get started by signing in with your preferred method
-            </CardDescription>
-          </div>
-        </CardHeader>
-        <CardContent className="space-y-4 pb-8">
-          <div className="space-y-4">
-            {providers.length > 0 ? (
-              providers.map((provider) => (
-                <ProviderButton
-                  key={provider.id}
-                  provider={provider}
-                  routeOnSuccess={routeOnSuccess}
-                />
-              ))
-            ) : (
-              <div className="text-center text-muted-foreground py-4">
-                No authentication providers available
-              </div>
-            )}
-          </div>
-        </CardContent>
-      </Card>
-      <AuthErrorBanner />
+    <div className="flex flex-col items-center justify-center min-h-20 w-full px-4">
+      <div className="w-full max-w-md space-y-8">
+        <div className="text-center space-y-3">
+          <h1 className="text-3xl md:text-5xl font-medium tracking-tight">
+            Welcome to <span className="italic font-sentient">tambo!</span>
+          </h1>
```

Comment:
This one is a little nuanced, but since the text includes the brand name, I'd prefer it to be the sans-serif font (suggested below) or the logotype (the version of the logo without the icon). Using the sentient font here is too close to the font used in the logotype but its not exact which adds visual confusion:

```suggestion
          <h1 className="text-3xl md:text-5xl font-medium tracking-tight">
            Welcome to Tambo
          </h1>
```